spec/03-plugins/17-jwt/03-access_spec.lua

```diff
@@ -150,6 +156,20 @@ describe("Plugin: jwt (access)", function()
       local json = cjson.decode(body)
       assert.same({ message = "No credentials found for given 'iss'" }, json)
     end)
+    it("returns 200 if the key is found in header", function()
+      local header = {typ = "JWT", alg = "HS256", kid = jwt_secret_2.key}
+      local jwt = jwt_encoder.encode(PAYLOAD, jwt_secret_2.secret, "HS256", header)
+      local authorization = "Bearer " .. jwt
+      local res = assert(proxy_client:send {
+        method  = "GET",
+        path    = "/request",
+        headers = {
+          ["Authorization"] = authorization,
+          ["Host"]          = "jwt10.com",
+        }
+      })
+      assert.res_status(200, res)
+    end)
```

Comment:
I don't think this test should be in the "refusals" section. The below test for "returns 200 on OPTIONS" either, but that is another issue.